server.ts

```diff
@@ -180,6 +179,15 @@ export function app() {
     changeOrigin: true
   }));
 
+  /**
+   * Proxy the linksets
+   */
+  router.use('/links**', createProxyMiddleware({
```

Comment:
When is this used?  This appears to be proxying `{dspace.ui.url}/links/{uuid}` to `{dspace.server.url}/signposting/links/{uuid}`.  But, strangely, I'm not seeing anywhere that we are using links which look like `{dspace.ui.url}/links/[uuid]`?

Reply:
Thanks, that answers my question then.  I understand why we'd need to proxy this so that the backend can always use links that start with `dspace.ui.url`.